src/components/Checkbox/Checkbox.types.ts

```diff
@@ -4,5 +4,5 @@ import { DefaultComponentProps, HTMLInputAttrs } from '../../types';
 
 export interface CheckboxProps extends DefaultComponentProps, HTMLInputAttrs {
 	checkIcon?: ReactNode;
-	label: string | ReactNode;
+	label?: string | ReactNode;
```

Comment:
Globaal even find & replace op `string | ReactNode` => `ReactNode`?

Reply:
Zal ik in een volgende PR fixen, string zit idd al mee in ReactNode 👍